### ISSUE TITLE
`EventHandler::mouse_enter_or_leave` is never called

### ISSUE THREAD

**Issue by IohannRabeson** [ggez/ggez#1183]
**Describe the bug**
The callback `EventHandler::mouse_enter_or_leave` is never called.

**To Reproduce**
Moving the mouse, when the cursor enters and leaves the GGez application window, the callback `EventHandler::mouse_enter_or_leave` is not called.

**Expected behavior**
I expect the callback be called when the mouse enters the window and when the mouse leaves the window

**Screenshots or pasted code**
This minimal example reproduce the issue, this application is supposed to print the text `mouse_enter_or_leave` on the standard output whenever the mouse enters or leaves the window. On my end, this method is never called.  

```rust
use ggez::*;

struct State {}

impl ggez::event::EventHandler<GameError> for State {
    fn update(&mut self, ctx: &mut Context) -> GameResult {
        Ok(())
    }
    fn draw(&mut self, ctx: &mut Context) -> GameResult {
        Ok(())
    }
    fn mouse_enter_or_leave(&mut self, _ctx: &mut Context, _entered: bool) -> Result<(), GameError> {
        println!("mouse_enter_or_leave");
        Ok(())
    }
}

pub fn main() {
    let state = State {};
    let c = conf::Conf::new();
    let (ctx, event_loop) = ContextBuilder::new("hello_ggez", "awesome_person")
        .default_conf(c)
        .build()
        .unwrap();
    event::run(ctx, event_loop, state)
}

```

**Hardware and Software:**
 - ggez version: 0.8.1
 - OS: MacOS
 - Graphics card: Intel Iris Pro 1536 MB
 - Graphics card drivers: -

**Comment by PSteinhaus** [comment]
Thanks for sharing.
The method is... actually never called in ggez at all...
I guess this just slipped through. Should be easy to add though. I think I'll give it a shot in the next days.
I just double-checked: the other EventHandler methods are actually all called somewhere... at least that ^^

**Comment by PSteinhaus** [comment]
Fixed in 07b8586 :)
Feel free to re-open if you still encounter problems.
